dbms/src/Common/Stopwatch.h

```diff
@@ -64,14 +64,14 @@ class Stopwatch
 
     void start()
     {
-        start_ns = nanoseconds();
+        start_ns = nanosecondsWithBound(start_ns);
```

Comment:
maybe better to use `steady_clock`, because as https://github.com/pingcap/tiflash/blob/master/dbms/src/Common/Stopwatch.h#L36 said, this `nanosecondsWithBound(start_ns)` may return 0

Reply:
but i'm ok with this implementation because I think the chance is small.

Reply:
I think `std::chrono::stead_lock` actually is a cross-platform wrap in C++ standard lib for `clock_gettime` with `CLOCK_MONOTONIC` in the Linux.

Reply:
https://github.com/llvm/llvm-project/blob/release/15.x/libcxx/src/chrono.cpp#L284-L293

Reply:
but we still have chance that `start_ns` is 0?